.github/workflows/release.yml

```
@@ -18,32 +18,40 @@ jobs:
-        with:
-          args: |
-            🚀 **Tailscale Release ${{ github.event.release.name }}**
-
-            View Release: ${{ github.event.release.html_url }}
-
-            **Changelog:**
-
-            ${{ github.event.release.body }}
+      # Print out the commit SHA
+      - name: Print commit SHA
+        run: echo "Commit SHA ${{ steps.release_action.outputs.commit_long_sha }}"
```

actionlint (1.7.7)

property "release_action" is not defined in object type {}

(expression)